{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "eb16590e_5f64bfd2",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 3
      },
      "lineNbr": 0,
      "author": {
        "id": 10366
      },
      "writtenOn": "2025-04-29T01:02:02Z",
      "side": 1,
      "message": "There are successful runs earlier without this change. Do you know why these errors are triggered.\n\nAlso please note from documentation the expectation is to have minimum number of avaialble IPs https://github.com/canonical/canonical-openstack-docs/pull/60/files",
      "revId": "1e127bcb7c5b67bed859a250e1c374f36204253c",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "d6733391_d908c9b4",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 3
      },
      "lineNbr": 0,
      "author": {
        "id": 10366
      },
      "writtenOn": "2025-04-29T01:49:08Z",
      "side": 1,
      "message": "tempestconf has similar logic already https://opendev.org/openinfra/python-tempestconf/src/commit/933cbff17b85e9e25dc58c08590422d60c4f60e6/config_tempest/services/network.py#L70",
      "parentUuid": "eb16590e_5f64bfd2",
      "revId": "1e127bcb7c5b67bed859a250e1c374f36204253c",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    }
  ]
}